pkg/backup/volume_snapshot_action.go

```diff
@@ -38,11 +39,15 @@ type volumeSnapshotAction struct {
 
 var _ Action = &volumeSnapshotAction{}
 
-func NewVolumeSnapshotAction(snapshotService cloudprovider.SnapshotService) Action {
+func NewVolumeSnapshotAction(snapshotService cloudprovider.SnapshotService) (Action, error) {
```

Comment:
I can see that... what I was thinking was "you're not going to call NewVolumeSnapshotAction if you don't have a snapshotService", but you never know what crazy things people will do 😄 . This is fine to keep.